### PR TITLE
Response

### PR DESCRIPTION
**Pull request by Nishkixs**

### CODE REVIEW

**Comment by Nishkixs** [comment]
Comfort: 3
Clarity: 3

File: diagnostic.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ Assume all code executes in strict mode.
 ## Question 1
 
 Please list the JavaScript primitives and give one example of each.
+1. Number  6
```
Comment (berziiii) [inline diagnostic.md:11]: Correct.

File: diagnostic.md [hunk 2]
```diff
@@ -16,10 +26,13 @@ let c = 5;
 let d = 2;
 c = c + d;
 
+
 ```
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
+Value of c is 7
+7 = 5 + 2
```
Comment (berziiii) [inline diagnostic.md:35]: Correct.

File: diagnostic.md [hunk 3]
```diff
@@ -33,7 +46,10 @@ y = 10;
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
+x=4
+y=3
 
+10 is reassigned to the value Y
```
Comment (berziiii) [inline diagnostic.md:52]: x = 3 from x = y thus reassigning x to the value of y which at the time was 3.

File: diagnostic.md [hunk 4]
```diff
@@ -45,6 +61,11 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+weather = "sunny"; This assigns "sunny" to weather
+
+weather === "sunny"; This is saying/asking if it is the same in both
+value and type. Essentially identical. so FALSE
```
Comment (berziiii) [inline diagnostic.md:67]: Values would be 

``` js
let weather; // undefined
weather = "sunny"; // "sunny"
weather === "sunny"; // true
```

reason being is that the VALUE of weather ("sunny") does infact strickly equal "sunny"

File: bin/diagnostic.js [hunk 5]
```diff
@@ -1 +1,15 @@
...
+
+let answer = 'SeCrEt';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer === 'SeCrEt') {
+   console.log("You guessed it!");
+   break;
+  }
+}
```
Comment (berziiii) [inline bin/diagnostic.js:15]: You needed to alter this. That was the intent of the question.

**Comment by berziiii** [comment]
3 / 5. This will sink in more as the week goes on.

**Pull request closed without merging**